data/src/main/kotlin/com/ortin/gesturetranslator/data/db/UserRoomDB.kt

```diff
@@ -0,0 +1,25 @@
+package com.ortin.gesturetranslator.data.db
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import com.ortin.gesturetranslator.data.entities.UserEntity
+
+@Database(
+    entities = [UserEntity::class],
+    version = 1
+)
+abstract class UserRoomDB: RoomDatabase() {
```

Comment:
```suggestion
abstract class UserInfoDatabase: RoomDatabase() {
```

Не нужно сокращать DB. Название должно отражать то, что хранится в этой базе

Reply:
done